### PR TITLE
feat(header): added ability to wrap logo in header component

### PR DESCRIPTION
**Pull request by korobko-virtru**
### Proposed Changes
_Please use the Jira Key followed by the changes_

- JIRAKEY-1234
  - Some change
  - Another change

### Checklist

- [x] I have added or updated unit tests
- [ ] I have added or updated integration tests (if appropriate)
- [ ] I have added or updated documentation / readme (if appropriate)
- [x] I have verified that my changes have not introduced new lint errors
- [x] I have added a GitHub labels to the PR for either _patch_, `minor`, or **major**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
+# v1.21.2 (Mon May 03 2021)
```
Comment (virtru-dubich) [inline CHANGELOG.md:1]: Date is not correct

File: lib/components/Layout/Header.css [hunk 2]
```diff
@@ -67,4 +66,8 @@
   display: none;
 }
 
+.branding {
+  padding-left: 24px;
```
Comment (virtru-dubich) [inline lib/components/Layout/Header.css:70]: we should use var(--vds...) instead of '24px'

File: lib/components/Layout/Header.js [hunk 3]
```diff
@@ -15,43 +15,48 @@ import styles from './Header.css';
...
 Branding.propTypes = {
   company: PropTypes.string,
   logo: PropTypes.oneOfType([PropTypes.node, PropTypes.func]),
+  logoRenderer: PropTypes.func,
 };
 
 Branding.defaultProps = {
   company: 'Virtru',
   logo: logoWhite,
+  logoRenderer: undefined,
```
Comment (virtru-dubich) [inline lib/components/Layout/Header.js:48]: maybe will be better to use null?

File: lib/components/Layout/Header.js [hunk 4]
```diff
@@ -74,13 +79,15 @@ Header.propTypes = {
...
   menuAlignmentDefault: PropTypes.oneOf(MENU_ALIGNMENTS_LIST),
+  logoRenderer: PropTypes.func,
 };
 
 Header.defaultProps = {
   company: 'Virtru',
   children: null,
   logo: logoWhite,
   menuAlignmentDefault: MENU_ALIGNMENTS_DEFAULT,
+  logoRenderer: undefined,
```
Comment (virtru-dubich) [inline lib/components/Layout/Header.js:90]: maybe will be better to use null?

File: lib/components/Layout/Header.css (hunk 2 above)
Comment (korobko-virtru) [reply lib/components/Layout/Header.css:70]: fixed

File: lib/components/Layout/Header.js (hunk 3 above)
Comment (korobko-virtru) [reply lib/components/Layout/Header.js:48]: null means that value still exists, while undefined tells that value is not specified at all

**Review by virtru-dubich — APPROVED** [review]

**Pull request merged**